_posts/2020-05-29-homebrew-2.3.0.md

```diff
@@ -0,0 +1,37 @@
+
+Major changes and deprecations since 2.2.0:
+
+- [`brew tap-pin` and `brew tap-unpin` have been removed along with the deprecation of some Homebrew and `Formula` methods.](https://github.com/Homebrew/brew/pull/7579)
+- [`brew install` from a URL has been deprecated to improve the security of `brew install`.](https://github.com/Homebrew/brew/pull/7660)
+- [`brew install`, `brew upgrade` and `brew reinstall` now fetch all resources before beginning installation](https://github.com/Homebrew/brew/pull/7124) or [locking dependencies](https://github.com/Homebrew/brew/pull/7635).
+- [The Homebrew/brew Docker image is now built for Ubuntu 16.04 and 20.04 and includes `latest` for the latest stable release and `master` for the `master` branch.](https://github.com/Homebrew/brew/pull/7617)
+- [Homebrew formulae can be deprecated or disabled to provide an easier transition than deletion (the previous method).](https://github.com/Homebrew/brew/pull/7190)
+- [Homebrew/homebrew-core has been entirely migrated to GitHub Actions](https://github.com/Homebrew/brew/issues/6255) and [Homebrew's Jenkins shut down entirely](https://github.com/Homebrew/brew/pull/7395).
+
+Other changes since 2.2.0 I'd like to highlight are the following:
+
+- [`brew style` and `brew readall` are run on macOS and Linux for Homebrew/brew and Homebrew/homebrew-core to make retaining Linux compatability easier](https://github.com/Homebrew/brew/pull/7469) and [ease the eventual merging of homebrew-core and linuxbrew-core taps](https://github.com/Homebrew/brew/issues/7028).
+- [Homebrew searches the entire PATH to find a compatible `ruby`.](https://github.com/Homebrew/brew/pull/7545)
+- [`brew cask info` shows cask analytics data.](https://github.com/Homebrew/brew/pull/7580)
+- [`brew doctor` shows deleted formulae.](https://github.com/Homebrew/brew/pull/7585)
+- [`brew uninstall` notes all `etc` files will stay around.](https://github.com/Homebrew/brew/pull/7526)
+- [`brew test` runs `pkill` without exceptions to avoid manual cleanup in `test do` blocks.](https://github.com/Homebrew/brew/pull/7368)
+- [`pkg-config` correctly sets the `SDKROOT` to find more macOS-provided software.](https://github.com/Homebrew/brew/pull/7277)
+- [Formulae can use the `pkgetc` method to install into `etc/#{formula_name}`.](https://github.com/Homebrew/brew/pull/7508)
+- [Formulae can use the `free_port` test helper.](https://github.com/Homebrew/brew/pull/7225)
+- [All Homebrew `curl` requests retry 2 times by default.](https://github.com/Homebrew/brew/pull/7196)
+- [Formulae `patch` blocks can change directory to apply their patch.](https://github.com/Homebrew/brew/pull/7132)
+- [`brew tap` defaults to full clones. The existing shallow clone default would cause slower `git fetch`s  over time.](https://github.com/Homebrew/brew/pull/6991)
```

Comment:
```suggestion
- [`brew tap` defaults to full clones. The existing shallow clone default would cause slower `git fetch`es  over time.](https://github.com/Homebrew/brew/pull/6991)
```
Up for debate: does the presence of backticks change the rules of pluralization?

Reply:
Not in my mind.